tests/dist.rs

```diff
@@ -32,11 +32,11 @@ use crate::mock::{MockComponentBuilder, MockFile, MockInstallerBuilder};
 
 const SHA256_HASH_LEN: usize = 64;
 
+// This is a test helper to edit the manifest.
+type EditFn<'a> = Option<&'a dyn Fn(&str, &mut MockChannel)>;
```

Comment:
Same name critique here as above - an EditFn is the parameter type to Option, the function *may take* an edit function, or not.